tests/Integration/Database/EloquentBelongsToManyTest.php

```diff
@@ -402,8 +403,8 @@ public function testFindOrNewMethod()
 
         $this->assertEquals($tag->id, $post->tags()->findOrNew($tag->id)->id);
 
-        $this->assertNull($post->tags()->findOrNew('asd')->id);
-        $this->assertInstanceOf(Tag::class, $post->tags()->findOrNew('asd'));
+        $this->assertNull($post->tags()->findOrNew(666)->id);
```

Comment:
PostgreSQL specifically fails here if you provide an incorrect type. Since the purpose is just to provide an invalid error, we can simply provide an invalid integer ID.